pkg/daemonset/daemonset.go

```diff
@@ -138,6 +138,71 @@ func (builder *Builder) WithNodeSelector(selector map[string]string) *Builder {
 	return builder
 }
 
+// WithHostNetwork applies HostNetwork to pod's definition.
+func (builder *Builder) WithHostNetwork() *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Enabling hostnetwork to daemonset %s in namespace %s",
+		builder.Definition.Name, builder.Definition.Namespace)
+
+	builder.Definition.Spec.Template.Spec.HostNetwork = true
+
+	return builder
+}
+
+// WithConfigMapVolume attaches given volume to all pod's containers.
```

Comment:
Please update func comment to something like:
// WithConfigMapVolume attaches given volume to all pod's containers under daemonset.

Reply:
updated as suggested